### ISSUE TITLE
NumberLiteral uses CurrentCulture.CurrentCulture

### ISSUE THREAD

**Issue by sebastienros** [daxnet/irony#4]
I'd like to understand when the behavior changed, looking at codeplex https://irony.codeplex.com/SourceControl/latest#Irony/Parsing/Terminals/NumberLiteral.cs it's using `CultureInfo.InvariantCulture` but in this repository it's `CultureInfo.CurrentCulture`.

The difference is that parsing a number like `123.45` on a French system for instance will fail as it's not using `.` as the decimal separator. Using `InvariantCulture` provides a consistent result at least, forcing `.`. 

Is that something that was introduced in the repository only? Or should we then change it to `InvariantCulture`?
Is there a more recent official repository other than the codeplex one?

**Comment by mattzink** [comment]
The README calls out that @daxnet changed to `CurrentCulture` on purpose. I agree with @sebastienros that this seems like this is incorrect and should be reverted back to `InvariantCulture`

**Comment by daxnet** [comment]
Using InvariantCulture is not applicable in .NET Core, as before .NET Core 2.0 there is no InvariantCulture defined in CultureInfo. 
I agree that using InvariantCulture is correct but it is not supported by .NET Core 1.1. I will update to .NET Core 2.0 and revert back this change.

**Comment by psylenced** [comment]
Perhaps you can use the NETCOREAPP2_0 conditional.

```
#if NETCOREAPP2_0
  // InvariantCulture
#else
  // CurrentCulture
#endif
```

**Comment by daxnet** [comment]
I just merged the pull request from @sebastienros, the current build that fixes this issue is 1.0.7.
I will take some time provide a .NET Standard 2.0 build side by side, once I get a chance to do that. Pretty busy these days....

**Comment by sebastienros** [comment]
If there is no reason to use netstandard 2.0, like a missing API, or a dependency that is only available on 2.0, then you don't have to.
